[MISC] Clarify participant_id in participants.tsv file if it exists (#738)
bids-standard/bids-specification: src/03-modality-agnostic-files.md

The purpose of this RECOMMENDED file is to describe properties of participants
such as age, sex, handedness.

> In case of single-session studies, if this file exists, this file MUST contain the column

Reviewer: ```suggestion
If this file exists, it file MUST contain the column
```

> `participant_id` that MUST consist of `sub-<label>` identifying each row for a

Reviewer 2: ```suggestion
`participant_id`, which MUST consist of `sub-<label>` values identifying one row for each
```

> If this file exists, it MUST contain the column
> `participant_id`, which MUST consist of `sub-<label>` values identifying one row for each
> participant, followed by a list of optional columns describing participants.

Reviewer: ```suggestion
If this file exists, it MUST contain the column `participant_id`,
which MUST consist of `sub-<label>` values identifying one row for each participant,
followed by a list of optional columns describing participants.
```
